x-pack/plugins/synthetics/server/synthetics_service/project_monitor_formatter.ts

```diff
@@ -228,17 +277,23 @@ export class ProjectMonitorFormatter {
     return savedObjects?.[0];
   };
 
-  private createMonitor = async (normalizedMonitor: BrowserFields) => {
-    await syncNewMonitor({
-      normalizedMonitor,
-      monitor: normalizedMonitor,
+  private createMonitorsBulk = async (normalizedMonitors: BrowserFields[]) => {
+    await syncNewMonitorBulk({
```

Comment:
All errors downstream are swallowed and opaque to the client. Can we improve this?

If we can catch the errors in an array, we can add them to the failed monitors key. 

Same for bulk updating and deletion.

Further, these bulk downstream tasks need to be wrapped in a try catch. I tested triggering downstream errors, and the final summary message containing `createdMonitors`, `failedMonitors`, etc, was never sent due to the unhandled error.